Loop experiments

sikorka opened the pull request:
Hello Miron 😄 Could you review? Thank you 👏

miron-kuzniar commented on gameproject/src/main/java/com/qatrain/janushgame/model/Board.java:
```diff
@@ -18,36 +18,36 @@
+        System.out.println("Creating board...");
 
-        rows = (int) (Math.random() * ROWS_MAX) + ROWS_MIN; //Random function from Math class returns the value between 0 and 1, so we have to use algorithm to create Grid with 3 <= size <= 16
+        rows = (int) (Math.random() * ROWS_MAX) + ROWS_MIN; //Random function from Math class returns the value between 0 and 1, so we have to use algorithm to create Board with 3 <= size <= 16
```
I just realized that this algorithm may give us values > than `ROWS_MAX` ie. (1 * 7) + 3 = 10. 
We should change this like this:
`rows = (int) (Math.random() * (ROWS_MAX - ROWS_MIN) + ROWS_MIN`

miron-kuzniar commented on gameproject/src/main/java/com/qatrain/janushgame/model/Board.java:
```diff
@@ -18,36 +18,36 @@
 
-        rows = (int) (Math.random() * ROWS_MAX) + ROWS_MIN; //Random function from Math class returns the value between 0 and 1, so we have to use algorithm to create Grid with 3 <= size <= 16
+        rows = (int) (Math.random() * ROWS_MAX) + ROWS_MIN; //Random function from Math class returns the value between 0 and 1, so we have to use algorithm to create Board with 3 <= size <= 16
         columns = (int) (Math.random() * COLUMNS_MAX) + COLUMNS_MIN; //The same as above
```
The same as above.

miron-kuzniar commented:
@sikorka 

Everything looks good except small change in Board:) Please check my code comment.

miron-kuzniar commented:
![](https://github.trello.services/images/mini-trello-icon.png) [As a user I can see Board](https://trello.com/c/VrOG3t5D/17-as-a-user-i-can-see-board)

sikorka commented:
Thank you for the review 👍  

Comments are applied + additional changes: 
- renamed module `gameproject` to `janushgame` 
- added packaging to jar in `pom.xml` + `README.md` and added how to run app from command line 
- commented `--` line printing `drawBoard()` so that shape of the board resembles dimensions 

Sorry for so many files changed. Could you pls review :)

miron-kuzniar commented:
@sikorka 

Great job. This game is awesome :D
Thanks!!!
